test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -1933,6 +1933,59 @@ Describe "Invoke-WebRequest tests" -Tags "Feature", "RequireAdminOnWindows" {
             $response = Invoke-WebRequest -Uri $dosUri
             $response.Images | Should -Not -BeNullOrEmpty
         }
+
+        It 'correctly parses an input tag with no end tag' {
+            $query = @{
+                contenttype = 'text/html'
+                body        = "<html><body><input name='foo' value='bar'></body></html>"
+            }
+            $uri = Get-WebListenerUrl -Test 'Response' -Query $query
+            $response = Invoke-WebRequest -Uri $uri -UseBasicParsing
+            $response.Error | Should -BeNullOrEmpty
+            $response.InputFields | Should -Not -BeNullOrEmpty
+            $response.InputFields.FindByName('foo') | Should -Not -BeNullOrEmpty
```

Comment:
Can we check exact result?

Reply:
Sure, I'll address this when I wrap up with my day job.